tests/models/embedding/language/test_embedding.py

```diff
@@ -49,3 +59,9 @@ def test_models(
     assert torch.all((all_similarities <= 1.0 + tolerance)
                      & (all_similarities >= 1.0 - tolerance)
                      ), f"Not all values are within {tolerance} of 1.0"
```

Comment:
```suggestion
    assert torch.all(all_similarities >= 1.0 - tolerance), f"Not all values are within {tolerance} of 1.0"
```

The first check is unnecessary since similarity values shouldn't exceed 1.